rate-providers/MorphoERC4626RateProviders.md

```diff
@@ -11,12 +11,17 @@
     - Steakhouse v1.1 USDC - csUSDC [ethereum:0xB0926Cfc3aC047035b11d9afB85DC782E6D9d76A](https://etherscan.io/address/0xB0926Cfc3aC047035b11d9afB85DC782E6D9d76A#code)
     - Gauntlet wETH Ecosystem v1.1[ethereum:0xD231564648C94542C01e9a528c9cAa033bbf274C](https://etherscan.io/address/0xD231564648C94542C01e9a528c9cAa033bbf274C#code)
     - IndexCoop mhyETH v1.1[ethereum:0x3BD4B2174498b3Aa01be0acFa0F775472b2dC30b](https://etherscan.io/address/0x3BD4B2174498b3Aa01be0acFa0F775472b2dC30b#code)
+    - Seamless USDC v1.1[base:0xc11082BbDBB8AaB12d0947EEAD2c8bc28E1b3B34](https://basescan.org/address/0xc11082BbDBB8AaB12d0947EEAD2c8bc28E1b3B34#code)
+    - Spark USDC v1.1[base:0x9e0926C3c5D2F42845Bf4F980F926b60323872d7](https://basescan.org/address/0x9e0926C3c5D2F42845Bf4F980F926b60323872d7#code)
+    - Ionic USDC v1.1[base:0xdCb03A77bB54E0a05D591e543FA39E9c46E8Febb](https://basescan.org/address/0xdCb03A77bB54E0a05D591e543FA39E9c46E8Febb#code)
+    - Ionic Ecosystem WETH v1.1[base:0xcc028cF7F8fA7986001fa7063A2E26b3094b42fd](https://basescan.org/address/0xcc028cF7F8fA7986001fa7063A2E26b3094b42fd#code)
```

Comment:
I pattern matched this info by looking at `SetIsAllocator` events, but I'm not sure it is all that meaningful since allocators can be added / removed by contract owner at any time, which means the info we are sharing here could easily be out of date

```
    function setIsAllocator(address newAllocator, bool newIsAllocator) external onlyOwner {
        if (isAllocator[newAllocator] == newIsAllocator) revert ErrorsLib.AlreadySet();

        isAllocator[newAllocator] = newIsAllocator;

        emit EventsLib.SetIsAllocator(newAllocator, newIsAllocator);
    }
```